src/utils/audio-normalizer.js

```diff
@@ -0,0 +1,89 @@
+// Inserts analyser and gain nodes after audio source.
+// Analyses audio source volume and adjusts gain value
+// to make it in a certain range.
+export class AudioNormalizer {
```

Comment:
Hard to compare the diff here since it was moved from another file. Is this just a direct copy/paste?

Reply:
Yes, this is just a copy paste of what live inside the avatar-volume-controls component.